packages/@okta/vuepress-site/docs/concepts/saml/index.md

```diff
@@ -121,9 +119,9 @@ A key consideration involves the ACS URL endpoint on the SP side where SAML resp
 
-In an SP-initiated flow, the user tries to access a protected resource directly on the SP side without the IdP being aware of the attempt. Two issues arise. First is the need to identify the right IdP if authentication of a federated identity is needed. With SP-initiated sign in, the SP initially doesn't know anything about the identity. As a developer, you need to figure out how the SP can determine which IdP should be receiving the SAML request. In some cases, if your application URLs contain subdomain information that is mapped to a unique tenant and IdP, then the resource link being hit is enough to identify the IdP. If this isn't the case, then you might need to prompt the end user for additional information from the end user such as user ID, email, or a company ID. You need something that allows the SP to identify which IdP the user attempting to access the resource belongs to. Remember, you are only prompting for an identifier, not credentials. Okta also supports passing the identifier to the IdP with parameter "LoginHint", so that the user doesn't need to input the identifier again when redirected to IdP to sign in. For instruction to trigger Okta to send the "LoginHint" to IdP, see [Redirecting with SAML Deep Links](/docs/reference/api/idps#redirecting-with-saml-deep-links).
+In an SP-initiated flow, the user tries to access a protected resource directly on the SP side without the IdP being aware of the attempt. Two issues arise. The first is the need to identify the right IdP if authentication of a federated identity is needed. With SP-initiated sign in, the SP initially doesn't know anything about the identity. As a developer, you need to figure out how the SP can determine which IdP should be receiving the SAML request. In some cases, if your application URLs contain subdomain information that is mapped to a unique tenant and IdP, then the resource link being hit is enough to identify the IdP. If this isn't the case, then you might need to prompt the end user for additional information from the end user such as user ID, email, or a company ID. You need something that allows the SP to identify which IdP the user attempting to access the resource belongs to. Remember, you’re only prompting for an identifier, not credentials. Okta also supports passing the identifier to the IdP with the parameter "LoginHint", so that the user doesn't need to input the identifier again when redirected to the IdP to sign in. For instruction to trigger Okta to send the "LoginHint" to IdP, see [Redirecting with SAML Deep Links](/docs/reference/api/idps#redirecting-with-saml-deep-links).
```

Comment:
I would put a colon after "Two issues arise:" and then bullet the two issues. This is just buried in a huge paragraph. I know you didn't write it - but it's a lot for the user to consume. 

```suggestion
In an SP-initiated flow, the user tries to access a protected resource directly on the SP side without the IdP being aware of the attempt. Two issues arise. The first is the need to identify the right IdP if authentication of a federated identity is required. With SP-initiated sign-in flows, the SP initially doesn't know anything about the identity. As a developer, you need to figure out how the SP can determine which IdP should be receiving the SAML request. In some cases, if your application URLs contain subdomain information that's mapped to a unique tenant and IdP, then the resource link being hit is enough to identify the IdP. If this isn't the case, then you might need to prompt the end user for additional information such as user ID, email, or a company ID. You need something that allows the SP to identify which IdP the user attempting to access the resource belongs to. Remember, you’re only prompting for an identifier, not credentials. Okta also supports passing the identifier to the IdP with the parameter "LoginHint", so that the user doesn't need to input the identifier again when redirected to the IdP to sign in. For instruction to trigger Okta to send the "LoginHint" to IdP, see [Redirecting with SAML Deep Links](/docs/reference/api/idps#redirecting-with-saml-deep-links).
```